tasks.py

```diff
@@ -777,3 +780,74 @@ def show_automerges(ctx: Context):
             print(f"{i}. @{pr['user']['login']} {pr['title']} {pr['html_url']}")
     else:
         print("\tNo PRs set to automerge")
+
+
+MARKER_REQ_MAPPING: Final[Mapping[str, tuple[str, ...]]] = {
+    "athena": ("reqs/requirements-dev-athena.txt",),
+    "cloud": ("reqs/requirements-dev-cloud.txt",),
+    "pyarrow": ("reqs/requirements-dev-arrow.txt",),
+    "external_sqldialect": ("reqs/requirements-dev-sqlalchemy.txt",),
+}
```

Comment:
Might tweak this mapping to also contain any necessary environmental or service setup needed for the test markers.